invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -94,20 +94,32 @@ def get_affiliation(self, obj):
                 }
                 identifiers = affiliation.get("identifiers")
                 if identifiers:
-                    # PIDS-FIXME: DataCite accepts only one, how to decide
-                    identifier = identifiers[0]
+                    # FIXME: Make configurable
+                    AFFILIATION_IDENTIFIER_TYPES_PREFERENCE = ("ror", "isni", "gnd")
+                    identifier = get_preferred_identifier(
+                        AFFILIATION_IDENTIFIER_TYPES_PREFERENCE, identifiers
+                    )
+                    if not identifier:
+                        identifier = identifiers[0]
+                        identifier["scheme"] = "Other"
+
                     id_scheme = get_scheme_datacite(
                         identifier["scheme"],
                         "VOCABULARIES_AFFILIATION_SCHEMES",
-                        default=identifier["scheme"],
-                    )
-
-                    if id_scheme:
-                        aff["affiliationIdentifier"] = identifier["identifier"]
-                        aff["affiliationIdentifierScheme"] = id_scheme.upper()
+                        default=identifier["scheme"].upper(),
                         # upper() is fine since this field is free text. It
                         # saves us from having to modify invenio-vocabularies
                         # or do config overrides.
+                    )
+
+                    if id_scheme == "ROR":
```

Comment:
I'm wondering if this is the case for more than one scheme, e.g. we also had ORCID URLs [here](https://github.com/inveniosoftware/invenio-rdm-records/commit/2cb3ecb5ae7a164a403442a64ce569fff8ec9b42#diff-e35eefcbf00ccd39e06e792ade38b37c8c0be576e979678bf099d1741e61ca0a), in which case we might need something more generic 🤔